app-emulation/vagrant/files/vagrant-1.9.7

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+#
+# This is a wrapper to properly execute Vagrant within the embedded
+# Vagrant installation directory. This sets up proper environmental variables
+# so that everything loads and compiles to proper directories.
+
+VAGRANT_DIR="$( ruby -e 'print Gem::default_path[-1] + "/gems/vagrant-1.9.7"' )"
+
+# Export GEM_HOME based on VAGRANT_HOME
+#
+# This needs to be set because Bundler includes gem paths
+# from RubyGems' Gem.paths.
+if [ -z $VAGRANT_HOME ]; then
+  VAGRANT_HOME=$(eval echo "~/.vagrant.d")
```

Comment:
What's this monster? Are you trying to force tilde substitution here? Isn't

    VAGRANT_HOME=~/.vagrant.d

much simpler? :-P